server/internal/handlers/pullrequest/edited.go

```diff
@@ -6,16 +6,17 @@ import (
 	"github.com/Mentro-Org/CodeLookout/internal/config"
 	ghclient "github.com/Mentro-Org/CodeLookout/internal/github"
 	"github.com/Mentro-Org/CodeLookout/internal/llm"
-
+    "github.com/jackc/pgx/v5/pgxpool"
 	"github.com/google/go-github/v72/github"
 )
 
 type PullRequestEditedHandler struct {
 	Cfg             *config.Config
 	AIClient        llm.AIClient
 	GHClientFactory *ghclient.ClientFactory
+	DB              *pgxpool.Pool
```

Comment:
```suggestion
DBPool *pgxpool.Pool
```
<img width="448" alt="image" src="https://github.com/user-attachments/assets/0dadbe90-9a9c-42b4-9dad-81d8e3905eae" />